package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@rsksmart/rsk-precompiled-abis",
-  "version": "4.0.2-IRIS",
+  "version": "4.0.3",
```

Please update the readme including the hop version in its compatibility matrix
```suggestion
  "version": "5.0.0-HOP",
```

---

Resolved

---

Should not the "Important note" section be updated too? I guess ~~ IRIS ~~ should be updated to HOP.

Currently text:
```
# Important note
If the version to be installed is not defined in the command line, the version will correspond to the latest version in IRIS.
```

---

Good Catch Nath. Updated To HOP